packages/amplify-category-auth/resources/lambda-function.js

```diff
@@ -0,0 +1,45 @@
+const response = require('cfn-response');
+const aws = require('aws-sdk');
+
+exports.handler = function (event, context) {
+  try {
+    const userPoolId = event.ResourceProperties.userpoolId;
+    const lambdaConfig = event.ResourceProperties.lambdaConfig;
+    const config = {};
+    lambdaConfig.forEach(lambda => (config[`${lambda.triggerType}`] = lambda.lambdaFunctionArn));
+    let promises = [];
+    if (event.RequestType == 'Delete') {
+      const authParams = { UserPoolId: userPoolId, LambdaConfig: {} };
+      const cognitoclient = new aws.CognitoIdentityServiceProvider();
+      promises.push(cognitoclient.updateUserPool(authParams).promise());
+      Promise.all(promises)
```

Comment:
need to await the promise. Also, would be better to turn this promise chain into async / await

Reply:
added await in promise